rules.d/60-fido-id.rules

```diff
@@ -11,4 +11,7 @@ SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ENV{ID_USB_INTERFACES}=="*:0b????:
 # Tag any CCID device (i.e. Smartcard Reader) as security token
 ENV{ID_SMARTCARD_READER}=="1", TAG+="security-device"
 
+# /dev/input/by-id/ symlink
+TAG=="security-device", ENV{ID_SERIAL}=="?*", SYMLINK+="input/by-id/%E{ID_SERIAL}"
```

Comment:
hmm, shouldn't we do this for all input devices then?

i.e. 60-persistent-input.rules already does this for mice and joysticks. It seems we should reuse the same rules but extend this to hidraw stuff genericlaly, no?

Reply:
That seems fine to me. I moved the rule there.